prow/plugins.yaml

```diff
@@ -72,112 +69,64 @@ approve:
   implicit_self_approve: true
 
 plugins:
-  istio/istio:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
-  istio/api:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
-  - verify-owners
-  - wip
-
-  istio/installer:
-  - trigger
+  istio:
+    - size
+    - trigger
+    - wip
 
-  istio/operator:
-  - assign
-  - hold
-  - slackevents
-  - trigger
-  - wip
+  istio/istio:
```

Comment:
We should either send a message to community or are the hold plugin back , this is used sometimes. Removing is fine with me though

Reply:
I would like to keep non-noisy, non-conflicting plugins like `assign`, `help`, `hold`, etc at least enabled where they are already.

People who like they can continue using them. People who do not do not need to bother with them.

If this is controversial can we move disabling them out of this PR so we can check in what we have in agreement?